trackers/react-native-tracker/src/events.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  buildPageView,
+  buildSelfDescribingEvent,
+  buildStructEvent,
+  PageViewEvent,
+  SelfDescribingJson,
+  StructuredEvent,
+  TrackerCore,
+} from '@snowplow/tracker-core';
+import { EventContext, MessageNotificationProps, TimingProps } from './types';
+
+export function newTrackEventFunctions(core: TrackerCore) {
```

Comment:
And an explicit return type here would be lovely.

Reply:
Same as https://github.com/snowplow/snowplow-javascript-tracker/pull/1371#discussion_r1836715416.